.github/workflows/testing.yml

```diff
@@ -38,6 +38,7 @@ jobs:
           sudo echo "127.0.0.1  nevermined-metadata" | sudo tee -a /etc/hosts
           git clone https://github.com/nevermined-io/tools
           cd tools
+          git checkout 992069f61b8961755bf4f26b5313a6d90ca4992e
```

Comment:
with latest nvm tools changes can we remove this?

Reply:
related to this https://github.com/nevermined-io/tools/pull/122
I'll remove it in the next pr